.travis.yml

```diff
@@ -4,9 +4,7 @@ node_js:
   - "10"
   - "9"
```

Comment:
Should we also drop Node 9 now that it's EOL?

Reply:
I was torn as ESLint v5 left node v9 in its test matrix, but I think that was because ESLint v5 was released JUST after node v9's EOL. (ESLint v5 release 2018-06-22, Node v9 was EOLed 2018-06-30).

I'm happy to drop support for it if you are.

Reply:
Sounds good to me, I don't think we need to optimize for users running EOL versions of Node.